source/retrieve-data.txt

```diff
@@ -0,0 +1,191 @@
+.. _java-rs-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code examples, read, search, cursor
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-long+} to retrieve
+data from a MongoDB collection by using read operations. 
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<java-rs-getting-started>`.
+
+.. include:: includes/reactor-note.rst
```

Comment:
Note: this note has changed. The preview is not accurate: when deployed in, this note will say:

.. important:: Project Reactor Library

   This guide uses the Project Reactor library to consume ``Publisher`` instances returned
   by the {+driver-short+} methods. To learn more about the Project Reactor library
   and how to use it, see `Getting Started <https://projectreactor.io/docs/core/release/reference/#getting-started>`__ in the Reactor documentation. To learn more about how we use Project Reactor
   library methods in this guide, see :ref:`Write Data to MongoDB <write-data-reactor-overview>`.
   
The reason this is not reflected right now is that this is an old branch made before the note changed. It will reflect in the final product.